### PR TITLE
Feature/aqm log

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

Redoing the logs for aqm and analyses, by using the same methods from other components and not moving the logfile to another directory for the purposes of cat'ting.  Now the cat'ting is done at the end of the exevs script.  

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

aqm:
Run the prep job.
Run the stats job.  Set COMIN to emc.vpppg.  Use qsub -v cyc=00, 04, 23 for these tests
Run the plots job. Set COMIN to emc.vpppg.  

analyses:
Run a stats job.  Use qsub -v cyc=00, 23 for these tests
Run a plots job.  Set COMIN to emc.vpppg.  

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA , for the stats jobs at vhr=23, should I force VDATE to be older so we test a case when all vhrs are available?

**Comment by PerryShafran-NOAA** [comment]
That's not necessary.  I just want to see that the logging works fine for all cases, but the size of the output files don't matter as long as one is formed.

**Comment by ShelleyMelchior-NOAA** [comment]
aqm prep and stats have completed. aqm plots is underway.
```
[dlogin08 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ lltr
total 6592
-rw-r--r-- 1 shelley.melchior emc  968944 Dec  6 18:08 jevs_aqm_prep_00.o120516036
-rw-r--r-- 1 shelley.melchior emc 1809251 Dec  6 18:08 jevs_aqm_grid2obs_stats.o120516094
-rw-r--r-- 1 shelley.melchior emc 2383567 Dec  6 18:09 jevs_aqm_grid2obs_stats.o120516098
-rw-r--r-- 1 shelley.melchior emc 1067796 Dec  6 18:15 jevs_aqm_grid2obs_plots.o120516189
-rw-r--r-- 1 shelley.melchior emc  510918 Dec  6 18:15 jevs_aqm_grid2obs_stats.o120516431
```

**Comment by PerryShafran-NOAA** [comment]
aqm prep looks good.

**Comment by PerryShafran-NOAA** [comment]
aqm stats look good too.

**Comment by ShelleyMelchior-NOAA** [comment]
analyses stats and plots are underway and/or completed.
```
[dlogin08 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ lltr jevs_analyses_*
-rw-r--r-- 1 shelley.melchior emc  550422 Dec  6 18:22 jevs_analyses_rtma_ru_grid2obs_stats.o120516662
-rw-r--r-- 1 shelley.melchior emc  591441 Dec  6 18:26 jevs_analyses_rtma_grid2obs_stats.o120516647
-rw-r--r-- 1 shelley.melchior emc  500380 Dec  6 18:26 jevs_analyses_urma_grid2obs_stats.o120516808
-rw-r--r-- 1 shelley.melchior emc  575221 Dec  6 18:26 jevs_analyses_rtma_ru_grid2obs_stats.o120516807
-rw-r--r-- 1 shelley.melchior emc  517453 Dec  6 18:26 jevs_analyses_rtma_grid2obs_stats.o120516806
-rw-r--r-- 1 shelley.melchior emc  549493 Dec  6 18:27 jevs_analyses_urma_grid2obs_stats.o120516715
-rw-r--r-- 1 shelley.melchior emc 1462655 Dec  6 18:27 jevs_analyses_grid2obs_plots.o120516922
```

**Comment by PerryShafran-NOAA** [comment]
The rtma_ru I think needs fixing.  Let me submit an update.

File: scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh [hunk 1]
```diff
@@ -11,10 +11,8 @@ set -x
 
 # Set up initial directories and initialize variables
 
-mkdir -p $DATA/plots
 mkdir -p $DATA/plots/logs
 export LOGDIR=$DATA/plots/logs
-export LOGFIN=$DATA/logs
 mkdir -p $LOGFIN
```
Comment (AliciaBentley-NOAA) [inline scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh]: @PerryShafran-NOAA @ShelleyMelchior-NOAA  I'm unsure how we can remove `export LOGFIN=$DATA/logs` and keep the following line (`mkdir -p $LOGFIN`). Can you please check this for me?

Comment (PerryShafran-NOAA) [reply scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh]: This is part of the problem.  I had moved the logfile to a new log directory, but that means that files were not in the original location as denoted by the METplus output.  LOGFIN should not be a directory.  Logs need to be kept in LOGDIR.

Comment (PerryShafran-NOAA) [reply scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh]: Removed LOGFIN.

**Comment by PerryShafran-NOAA** [comment]
@ShelleyMelchior-NOAA Please re-run the analyses plots.  Before you do that, please delete the plot output directory.

**Comment by ShelleyMelchior-NOAA** [comment]
analyses plots has been resubmitted, following removal of the analyses plots COMOUT directory.
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_grid2obs_plots.o120519335`
Do I need to re-run some analyses stats, too?

**Comment by PerryShafran-NOAA** [comment]
Yes, please re-run all the analyses stats jobs.

**Comment by ShelleyMelchior-NOAA** [comment]
analyses stats have been resubmitted for vhr=00 and vhr=23.
```
[dlogin08 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ lltr jevs_analyses_*stats*.o1205206*
-rw-r--r-- 1 shelley.melchior emc 504165 Dec  6 19:22 jevs_analyses_rtma_grid2obs_stats.o120520660
-rw-r--r-- 1 shelley.melchior emc 500380 Dec  6 19:22 jevs_analyses_urma_grid2obs_stats.o120520643
-rw-r--r-- 1 shelley.melchior emc 660578 Dec  6 19:23 jevs_analyses_rtma_ru_grid2obs_stats.o120520642
-rw-r--r-- 1 shelley.melchior emc 500380 Dec  6 19:23 jevs_analyses_urma_grid2obs_stats.o120520662
-rw-r--r-- 1 shelley.melchior emc 517453 Dec  6 19:23 jevs_analyses_rtma_grid2obs_stats.o120520641
-rw-r--r-- 1 shelley.melchior emc 697210 Dec  6 19:23 jevs_analyses_rtma_ru_grid2obs_stats.o120520661
```

**Comment by PerryShafran-NOAA** [comment]
There are still issues with this in the analyses stats.  I need to move the block of code.

**Comment by PerryShafran-NOAA** [comment]
Remove the stats output directory and run the analyses jobs again please.

**Comment by ShelleyMelchior-NOAA** [comment]
analyses stats have been re-resubmitted for vhr=00 and vhr=23.
```
[dlogin08 /lfs/h2/emc/ptmp/shelley.melchior/EVS_out]$ lltr jevs_analyses_*stats*.o1205214*
-rw-r--r-- 1 shelley.melchior emc 487598 Dec  6 19:38 jevs_analyses_urma_grid2obs_stats.o120521464
-rw-r--r-- 1 shelley.melchior emc 487660 Dec  6 19:39 jevs_analyses_urma_grid2obs_stats.o120521468
-rw-r--r-- 1 shelley.melchior emc 504165 Dec  6 19:39 jevs_analyses_rtma_grid2obs_stats.o120521462
-rw-r--r-- 1 shelley.melchior emc 504165 Dec  6 19:39 jevs_analyses_rtma_grid2obs_stats.o120521466
-rw-r--r-- 1 shelley.melchior emc 661713 Dec  6 19:39 jevs_analyses_rtma_ru_grid2obs_stats.o120521467
-rw-r--r-- 1 shelley.melchior emc 638419 Dec  6 19:40 jevs_analyses_rtma_ru_grid2obs_stats.o120521463
```

**Comment by PerryShafran-NOAA** [comment]
Looking good for rtma_ru!  rtma and urma take a bit longer but I'm assuming if it works for rtma_ru it'll work for urma and rtma too.

**Comment by PerryShafran-NOAA** [comment]
aqm plots finished and also looks good.

**Comment by PerryShafran-NOAA** [comment]
rtma stats look good.

**Comment by PerryShafran-NOAA** [comment]
And urma stats look good too!

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA @ShelleyMelchior-NOAA Yay! I have looked over and commented on the changed files, and I think they all look good now! I approve merging this PR.

**Comment by ShelleyMelchior-NOAA** [comment]
I will perform formal code review.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Tests produce anticipated results. This PR is approved for merge.

**Pull request merged**